src/parquet/column/test-specialization.h

```diff
@@ -115,7 +115,8 @@ void PrimitiveTypedTest<TestType>::SyncValuesOut() {}
 
 template <>
 void PrimitiveTypedTest<BooleanType>::SyncValuesOut() {
-  std::copy(bool_buffer_out_.begin(), bool_buffer_out_.end(), values_out_.begin());
+  std::transform(bool_buffer_out_.begin(), bool_buffer_out_.end(), values_out_.begin(),
```

Comment:
Change made to overcome MSVC warning [c4800](https://msdn.microsoft.com/en-us/library/b6801kcy.aspx). Warning happens on assigning of bool to non-boolean value. Type casting doesn't affect appearance of this warning.
As alternative solution, we can just suppress this kind of warnings, like this done in some open source projects.